test/AutoRest.TestServer.Tests/Common/Generation/Writers/ModelWriterTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is a new test class that I added.

Reply:
If these are unit tests they should go in a unit test project.

This project is supposed to be kept for testing the resulting shape of all autorest.TestServer projects

Reply:
fixed.